yml/apdb.yaml

```diff
@@ -2186,12 +2201,14 @@ tables:
     mysql:datatype: DATETIME
   primaryKey:
   - "#DiaForcedSource.diaObjectId"
-  - "#DiaForcedSource.ccdVisitId"
+  - "#DiaForcedSource.detector"
+  - "#DiaForcedSource.visit"
   indexes:
-  - name: IDX_DiaForcedSource_ccdVisitId
-    "@id": "#IDX_DiaForcedSource_ccdVisitId"
+  - name: IDX_DiaForcedSource_detectorVisit
+    "@id": "#IDX_DiaForcedSource_detectorVisit"
     columns:
-    - "#DiaForcedSource.ccdVisitId"
+    - "#DiaForcedSource.detector"
+    - "#DiaForcedSource.visit"
```

Comment:
Change the order to visit, detector. May not be very important, but it's better to keep it consistent with DiaSource. Also it can be useful for queries like `SELECT * from DiaForcedSources WHERE visit = 1000`. I don't think that query `SELECT * from DiaForcedSources WHERE detector = 10` is useful.

Reply:
We're definitely more likely to search on visit, but I suspect searches on just detector will be useful for debugging, e.g. "does detector 10 always have some kind of failure mode in its sources?"

Reply:
If those queries are important/frequent, then maybe worth adding an extra index? I guess we'll need a systematic approach to figure out performance issues later.